.travis.yml

```diff
@@ -5,4 +5,6 @@ notifications:
 
 install: npm ci
```

Comment:
this is not installing and throwing an error
https://travis-ci.com/github/IBM/exported-tests/builds/154821209#L205
```
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's canvas@2.6.1 does not satisfy canvas@2.5.0
npm ERR! 
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-03-24T16_11_29_839Z-debug.log
The command "npm ci" failed and exited with 1 during .
```

Please re-create the lock file